### ISSUE TITLE
Horizontal History - Wait But Why

### ISSUE THREAD

**Issue by guevara** [guevara/read-it-later#1222]
Horizontal History - Wait But Why<br>
<br>
https://ift.tt/2uYxVgC<br>
<br>
About Tim Urban View all posts by Tim Urban &rarr;<br>
<br>
<div>
<div>
<p>Most of us have a pretty terrible understanding of history. Our knowledge is spotty, with large gaps all over the place, and the parts of history we do end up knowing a lot about usually depend on the particular teachers, parents, books, articles, and movies we happen to come across in our lives. Without a foundational, tree-trunk understanding of all parts of history, we often forget the things we do learn, leaving even our favorite parts of history a bit hazy in our heads. Raise your hand if you&rsquo;d like to go on stage and debate a history buff on the nuances of a historical time period of your choosing. That&rsquo;s what I thought.</p>
<p>The reason history is so hard is that it&rsquo;s so <em>soft</em>. To truly, fully understand a time period, an event, a movement, or an important historical figure, you&rsquo;d have to be there, and many times over. You&rsquo;d have to be in the homes of the public living at the time to hear what they&rsquo;re saying; you&rsquo;d have to be a fly on the wall in dozens of secret, closed-door meetings and conversations; you&rsquo;d need to be inside the minds of the key players to know their innermost thoughts and motivations. Even then, you&rsquo;d be lacking context. To really have the complete truth, you&rsquo;d need background&mdash;the cultural nuances and national psyches of the time, the way each of the key players was raised during childhood and the subtle social dynamics between those players, the impact of what was going on in other parts of the world, and an equally-thorough understanding of the many past centuries that all of these things grew out of.</p>
<p>That&rsquo;s why not only can&rsquo;t even the most perfect history buff fully understand history, but the key people involved <em>at the time</em> can&rsquo;t ever know the full story. History is a giant collective tangle of thousands of interwoven stories involving millions of characters, countless chapters, and many, many narrators.</p>
<p>And you know humans&mdash;that&rsquo;s not how they like things. The human brain really, really likes to <em>simplify</em> things. History provides the context of our world and our lives, because each of us is a character in this grand story&mdash;and the last thing we want to believe is that the story is too complicated and mysterious for us to understand.</p>
<p>Fairy tales are satisfying, because the plot is crystal clear&mdash;there are good guys and there are bad guys and there&rsquo;s only one side of the story. Children identify with the good guys&mdash;the <em>us</em> guys&mdash;and they detest the bad guys&mdash;the <em>them</em> guys<em>&mdash;</em>and everyone&rsquo;s happy. Stories written for adults aren&rsquo;t that different&mdash;you loved <em>Shawshank</em> and <em>Braveheart</em> and <em>Star Wars</em>, right?</p>
<p>So when it comes to the story we&rsquo;re all a <em>part</em> of, we most certainly want to feel the same way. We want history to be simple and clear, with good guys and bad guys, and we&rsquo;d like to make sure that our ancestors, our ethnic group, our nation, and all the other tribes we belong to are Aladdin in the story&mdash;not Jafar.</p>
<p>The problem with this is that not everyone can be Aladdin. <em>Someone</em> has to be Jafar, right? Well, no. Not if there are many different story-tellers. Since no one is ever telling anything close to the full, real, <em>complete</em> story, in all its complexity&mdash;as we said, no one even <em>knows</em> the full story&mdash;each historian, each ruler, and each society creates their own fairy tale version of what went down in the past. When things are unsatisfyingly multi-faceted, we pick the facet we like best. When there are knowledge gaps, we make things up. When there are questions of motive, we pick one that fits nicely into the narrative.</p>
<p>This leaves us with plenty of tools to leave every story with a proper Aladdin and a proper Jafar and allows us to make sure that Aladdin is exactly who we want him to be.</p>
<p>The US is a good example. A huge number of people in today&rsquo;s world have been told a story of the US in which the US is Aladdin, and a huge other number of people have heard the same story with the US as Jafar. Some people will claim to have a more nuanced view, but deep in their heart, when they see an American flag, they see either a good guy flag or a bad guy flag. (One of the major political divides in the US stems from liberals thinking conservatives over-Aladdinize the US and conservatives thinking liberals over-Jafarify the US while Aladdinizing the other side.)</p>
<p>This is the same phenomenon behind the stark opinion divide around Israel and Palestine. Hordes of people on both sides of what is an <em>insanely</em> complicated story are red in the face with ire at the other side, completely positive that their side is Aladdin and incensed that anyone could ever call the other side Aladdin and their side Jafar. Only with the stark clarity of a fairy tale could people ever feel so unshakingly sure.</p>
<p>Of course, it&rsquo;s not that there are no good guys or bad guys in history. History is a pretty ugly story&mdash;what else would you expect from a species of primitive biological animals&mdash;and accountability for that ugliness isn&rsquo;t spread out evenly amongst all people. To an extent, the definition of words like good and bad, right and wrong, and hero and villain lie in the eye of the beholder&mdash;but there&rsquo;s also plenty of human behavior that qualifies as objectively good or bad.</p>
<p>So it&rsquo;s not that there are never objective Aladdins and there are never objective Jafars&mdash;it&rsquo;s that almost none of us has any idea what the fuck we&rsquo;re talking about. Point to a historical event and tell me that there was a true Aladdin and Jafar going on, and I&rsquo;ll acknowledge that that might be true. Tell me that you know who was who, and in most cases I&rsquo;ll shake my head.</p>
<p>Which brings me to me. Blogging about history is asking for trouble. Portray nearly any story or person as an Aladdin or a Jafar and you&rsquo;ll feel the wrath of both the people who believe the opposite situation and the people who think you&rsquo;ve oversimplified the situation. Portray something in a nuanced and balanced way and you&rsquo;ll get yelled at by people who believe <em>both</em> of the one-sided views. Nothing brings people&rsquo;s tribal fires to the table like history. I&rsquo;ve learned this <a href="https://waitbutwhy.com/2014/09/muhammad-isis-iraqs-full-story.html" target="_blank">from</a> <a href="https://waitbutwhy.com/2014/05/absurdly-famous-people-probably-dont-know-enough.html" target="_blank">experience</a>.</p>
<p>This doesn&rsquo;t make me any less excited to write about history&mdash;but it makes me want to research the shit out of a part of history before I write about it. Only by reading a bunch of varying accounts and opinions can you start to form a clear picture of what we know and don&rsquo;t know.</p>
<p>So that&rsquo;s why for this post, I&rsquo;m not gonna tell you shit. Rather than dive into the weeds of what happened when, and why, I&rsquo;m going to focus on one of the rare elements of history that&rsquo;s indisputably black-and-white&mdash;<em>who</em> happened when.</p>
<p>Because before we can responsibly start arguing with each other about Aladdins and Jafars, we need to get the basic timeline and characters of the story clear.</p>
<p>But I&rsquo;m going to lay things out a little differently than you&rsquo;re probably used to.</p>
<p>Normally, we learn about history&rsquo;s storylines in isolation. We might have a strong sense of the history of physics breakthroughs or the progression of western philosophical thought or the succession of French rulers&mdash;but we&rsquo;re not as clear on how each of these storylines relate to each <em>other</em>. If you think of history like a tangle of vines growing upwards through time, studying one type of history at a time is like following the path of one particular vine while ignoring the other vines around it. It&rsquo;s understanding history in a <em>vertical</em> sense.</p>
<p>And while vertical history has its merits, it doesn&rsquo;t leave you with an especially <em>complete</em> picture of any one time. An econ buff in the year 2500 might know all about the Great Depression that happened in the early 20th century and the major recession that happened about 80 years later, but that same person might mistake the two world wars for happening in the 1800s or the 2200s if they&rsquo;re a little hazy on the history of wars. So while an econ buff, that person would have a pretty poor understanding of what our modern times are all <em>about.</em></p>
<p>Likewise, I might know that Copernicus began writing his seminal work <i>On the Revolutions of the Celestial Spheres</i> in Poland in the early 1510s, but by learning that right around that same time in Italy, Michelangelo painted the ceiling of the Sistine Chapel, I get a better picture of the <em>times</em>. By learning that it was right while both of these things were happening that Henry VIII married Catherine of Aragon in England, the 1510s suddenly begins to take on a distinct <em>personality</em>. These three facts, when put together, allow me to see a more three-dimensional picture of the 1510s&mdash;it allows me to see the 1510s <em>horizontally</em>, like cutting out a complete segment of the vine tangle and examining it all together.</p>
<p>A blog post is limited in its ability to examine all of history horizontally. But I&rsquo;ve taken two separate cracks below that I think can work together nicely to help us take a horizontal view of different times. Both involve a lot of names.</p>
<p>Which leads me to the inevitable disclaimer about <em>who</em> I chose to include. I tried to remove my own biases by gathering the names from a handful of lists by publications like <em>Time</em>. I searched the internet for things like &ldquo;most influential people in history&rdquo; and &ldquo;most important people in the Middle Ages&rdquo; and &ldquo;most famous people of the 19th century&rdquo; and &ldquo;most powerful Chinese emperors&rdquo; and ended up with a big pile of names, some of whom I&rsquo;m familiar with, others I&rsquo;m not. That said, between the fact that the lists I used were by publications targeting English-speaking people and that I inevitably leaned more towards people I had heard of, the group of names will skew America- and Euro-centric, with places like Africa, the Middle East, and Southeast Asia probably underrepresented. This isn&rsquo;t entirely by accident, though&mdash;this post is only useful if you&rsquo;ve heard of the people, and I intentionally chose names I thought a large portion of Wait But Why readers would know. In other words, merit wasn&rsquo;t the only criteria&mdash;household fame mattered too. And yes, I missed a lot of people&mdash;with limited space on the screen, the names had to be a sampling, not an exhaustive list.</p>
<p><span><strong>Horizontal History&mdash;First Crack:</strong></span></p>
<p>For my first crack, I present to you a big pile of famous names, organized by birth decade&mdash;kind of a &ldquo;2,600 Under 2,600&rdquo; list. The purpose is to help orient ourselves on when people lived, especially <em>in relation to each other</em>.</p>
<p>Having a clear picture of generations is very easy when you think about currently-living people. For example, I know that Mark Zuckerberg is around my age while Vladimir Putin is about the age of my parents and George H.W. Bush is about the age of my grandparents. On the other side of things, Prince George&mdash;the one world-famous baby&mdash;is the age of my kids if I had kids. I know this without having to think about it:</p>
<p><a href="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/My-generations.png" target="_blank" rel="attachment wp-att-4675"><img src="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/My-generations.png" alt="My generations"></a></p>
<p>If I list people by birth decade instead of generation, it still makes sense. People born in the 70s and 60s feel older than me but not as old as my parents, and people born in the 30s and 40s feel older than my parents but younger than my grandparents:</p>
<p><a href="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/My-generations-decades.png" target="_blank" rel="attachment wp-att-4674"><img src="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/My-generations-decades.png" alt="My generations - decades"></a></p>
<p>But this is much harder for generations that aren&rsquo;t currently alive, and it gets less and less clear the farther back you go. Quick! Name the oldest member and youngest member of this group: Nietzsche, Darwin, Freud, Marx, Gandhi, Tolstoy, Twain. Not that easy, right? And that&rsquo;s only going back 200 years. But by laying them out by birth decade, you can get oriented:</p>
<p><a href="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/Older-decades.png" target="_blank" rel="attachment wp-att-4642"><img class="wp-image-4642 aligncenter" src="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/Older-decades.png" alt="Older decades"></a></p>
<p>Since a generation is typically about 30 years, you can move three or six lines down from a name to see who they viewed as their parents&rsquo; or grandparents&rsquo; ages during their lifetimes, and you can go the same distance up to see who they viewed as their kids&rsquo; or grandkids&rsquo; age. So Darwin would have seen Twain as some young kid and he would have shaken his old man fist at Gandhi from the rocking chair on his porch. Meanwhile, Nietzsche would have seen Marx as a guy his dad&rsquo;s age and Freud as a contemporary, though a bit younger.</p>
<p>Two people more than seven or eight lines away from each other on the list probably were not ever alive at the same time, which means they were likely not that clear about each others&rsquo; generation, in the way I&rsquo;m not really clear on whether Hemingway was in my great-grandparents&rsquo; generation, my great-great-grandparents&rsquo; generation, or some other age.</p>
<p>Using this decade list tool, let&rsquo;s look at a whole group of famous historical figures to see who was the same age as whom, who shook their old man fist at whom, and who was and wasn&rsquo;t alive at the same time. The decade colors are in a three-way cycle, so you can jump to rows of the same color above and below to quickly go up and down by generations (i.e. if you take a name on a green line, one green line down is their parents&rsquo; age, three green lines up is their great-grandchildren&rsquo;s age, etc.). For people alive today and in the past century, I couldn&rsquo;t come close to including every famous person, so I just picked a sampling.</p>
<p><a href="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/Horizontal-History-Generations-4.jpg" target="_blank" rel="attachment wp-att-4679"><img src="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/Horizontal-History-Generations-4.jpg" alt="Horizontal History - Generations"></a></p>
<p>Okay how did that go? Fun? Icky? I can&rsquo;t quite tell. In any case, let&rsquo;s move on.</p>
<p><span><strong>Horizontal History&mdash;Second Crack:</strong></span></p>
<p>For my second crack at depicting history horizontally, I present to you A Psychotic Jumble of Colorful Vertical Bars That Might Be Awesome or Maybe Not I Can&rsquo;t Tell Here Either But I Think It Might Be Fun.</p>
<p>This time, I got more specific than birth decade and actually identified the exact birth year <em>and</em> the death year of each name, using a bar to depict their lifespan. While the above chart simplifies who lived when, the diagram below allows you to follow a single horizontal path along any year and see who was and wasn&rsquo;t alive at that time.</p>
<p>As I made the diagram, I ran into a big problem, which is that it looked like an upside-down L with <em>way</em> more names at the top than the bottom (because there are a lot more household names who were alive in the last 200 years than in previous centuries). Crunching all those recent names into blog width made the font tiny&mdash;so I solved the problem by cutting out about half of the recent names. <em>But</em>, before I did, I broke the complete list of 1800&ndash;2016 names into two groups by category and here they are below:</p>
<p><a href="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/Horizontal-History-Lifespans-Recent-1-2.jpg" target="_blank" rel="attachment wp-att-4682"><img src="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/Horizontal-History-Lifespans-Recent-1-2.jpg" alt="Horizontal History - Lifespans (Recent 1)"></a></p>
<p><a href="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/Horizontal-History-Lifespans-Recent-2-2.jpg" target="_blank" rel="attachment wp-att-4683"><img src="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/Horizontal-History-Lifespans-Recent-2-2.jpg" alt="Horizontal History - Lifespans (Recent 2)"></a></p>
<p>Okay now that that&rsquo;s out of the way, here&rsquo;s the big list (with only half of the total 1800&ndash;2016 names included). It goes back to 1450. Trace a horizontal line across to get a feel for what was going on during that particular time.</p>
<p><a href="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/Horizontal-History-Lifespans-4.jpg" target="_blank" rel="attachment wp-att-4684"><img src="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/Horizontal-History-Lifespans-4.jpg" alt="Horizontal History - Lifespans"></a></p>
<p>[</p>
<strong>Note:</strong>
<p>A number of people have requested a sideways version of this diagram.</p>
<a href="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/Horizontal-History-Lifespans-Sideways.jpg" target="_blank">Here it is.</a>
<p>]</p>
<p>Some overall thoughts:</p>
<ul>
<li>
<strong>Lifespans are unfair.</strong> Looking at people&rsquo;s lives visually really makes it clear when two people are born around the same time but then one of them randomly dies 30 years before the other.</li>
</ul>
<ul>
<li>
<strong>Murder is dickish.</strong> Another thing this diagram highlights. How not okay is it to cut someone <em>else&rsquo;s</em> bar short? JFK might have been on his way to a nice 85-year bar when this other guy just took a scissors and <em>snipped</em> his bar.</li>
</ul>
<ul>
<li>
<b>On the other hand, short lives were appreciated during the making of this diagram.</b> This was a nightmare of a puzzle, especially at the top, and while trying to fit a lot of bars into a small space, there were times I found myself saying, out loud, &ldquo;Oh nice,&rdquo; when I&rsquo;d look up someone&rsquo;s life dates and realize that they were murdered at a young age. Likewise, one factor that led to a number of the recent people being cut from the big diagram was living too long. Frank Lloyd Wright&rsquo;s a cool dude, but not 2.5 inches of diagram cool.</li>
</ul>
<ul>
<li>
<strong>Some people aren&rsquo;t easily categorized<em>.</em></strong> I tried my best. You try putting Ben Franklin into a category.</li>
</ul>
<ul>
<li>
<strong>Yeah, yeah, I said the whole &ldquo;Aladdin and Jafar are in the eye of the beholder&rdquo; thing and then I created a category for people who I deemed dicks.</strong>I know. But it was fun to label certain people as dicks. Ya know?</li>
</ul>
<p>Each little part of this diagram tells a story. Let&rsquo;s go through a few examples:</p>
<p>I mentioned in a box in the first chart that Mozart wrote his <em>Requiem</em> the same year the US forefathers were writing the Bill of Rights and that Beethoven had a love-hate relationship with Napoleon&mdash;but using the lifespan diagram, you can see both of these stories visually.</p>
<p><a href="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/1780.jpg" target="_blank" rel="attachment wp-att-4651"><img class="wp-image-4651 aligncenter" src="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/1780.jpg" alt="1780"></a></p>
<p>I also mentioned the major Shakespeare, Galileo, Tokugawa, and John Smith events that all happened right around the year 1610.</p>
<p><a href="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/1610.png" target="_blank" rel="attachment wp-att-4686"><img src="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/1610.png" alt="1610"></a></p>
<p>And in the intro, I referenced Copernicus&rsquo;s seminal work happening right when Michelangelo was painting the Sistine Chapel ceiling and Henry VIII was marrying Catherine of Aragon.</p>
<p><a href="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/1510.png" target="_blank" rel="attachment wp-att-4685"><img src="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/1510.png" alt="1510"></a></p>
<p>But if you look at the other stuff the diagram shows going on around that same time, it tells a bigger story:</p>
<p><a href="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/Renaissance-2.jpg" target="_blank" rel="attachment wp-att-4680"><img class="wp-image-4680 aligncenter" src="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/Renaissance-2.jpg" alt="Renaissance"></a></p>
<p>While Copernicus, Michelangelo, and Henry VIII were happening, it was also the golden age of European exploration&mdash;look at all those light purple explorers!&mdash;and the precursor age to the oncoming Age of Imperialism. Simultaneously, you can see the Protestant Reformation brewing with the presence of all those dark blue religious figures. The one dark blue exception is Guru Nanak, who was over in Asia being the founding prophet of Sikhism (today&rsquo;s 5th biggest religion). Meanwhile, Michelangelo was part of something larger, as the other pink bars&mdash;and Machiavelli&mdash;remind us that the Italian Renaissance was in full swing.</p>
<p>Every time I look at the lifespan diagram, a new interesting horizontal pops out to me. Here&rsquo;s one more: People in the US associate the 1860s with Lincoln and the Civil War. But what we overlook is that the 1860s was one of history&rsquo;s greatest literary decades. In the ten years between 1859 and 1869, Darwin published his world-changing <em>On the Origin of Species</em> (1859), Dickens published <em>A Tale of Two Cities</em> (1859) and <em>Great Expectations</em> (1861), Lewis Carroll published <em>Alice in Wonderland</em> (1865), Dostoyevsky published <em>Crime and Punishment</em> (1866), and Tolstoy capped things off with <em>War and Peace</em> (1869). These guys were all in their primes <em>at the same time</em>. So was Lincoln, before some cock snipped his bar off at the worst time possible.</p>
<p><a href="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/1860s-2.jpg" target="_blank" rel="attachment wp-att-4666"><img class="wp-image-4666 aligncenter" src="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/uploads/2016/01/1860s-2.jpg" alt="1860s"></a></p>
<p>So there&rsquo;s some horizontal history for you. Now go brush up so we can all be oriented the next time we yell at each other about fairy tales.</p>
<p>___________</p>
<p>If you&rsquo;re into Wait But Why, sign up for the <strong><a href="http://eepurl.com/Ffj9D" target="_blank">Wait But Why email list</a></strong> and we&rsquo;ll send you the new posts right when they come out. Better than having to check the site and wonder!</p>
<p>If you&rsquo;d like to support Wait But Why, <strong><a href="https://www.patreon.com/waitbutwhy" target="_blank">here&rsquo;s our Patreon</a></strong>.</p>
<p>___________</p>
<p><span><strong>More Wait But Why history posts:</strong></span></p>
<p><strong><a href="https://waitbutwhy.com/2014/09/muhammad-isis-iraqs-full-story.html" target="_blank">From Muhammad to ISIS: Iraq&rsquo;s Full Story</a></strong> &ndash; written after my visit to Kurdish Iraq</p>
<p><a href="https://waitbutwhy.com/2013/08/putting-time-in-perspective.html" target="_blank"><strong>Putting Time in Perspective</strong></a> &ndash; a diagram to help visualize everything that happened between the Big Bang and now</p>
<p><a href="https://waitbutwhy.com/2013/08/the-death-toll-comparison-breakdown.html" target="_blank"><strong>The Death Toll Comparison Breakdown</strong></a> &ndash; a diagram to put famous death tolls in perspective</p>
<p><strong><a href="https://waitbutwhy.com/2014/02/american-presidents-washington-lincoln.html" target="_blank">The American Presidents series</a></strong></p>
<div>
<h3>RECOMMENDED POSTS</h3>
<ul>
<li>
<li>
<p><span>April 20, 2017</span></p>
</li>
<li>
<li>
<p><span>March 24, 2016</span></p>
</li>
<li>
<li>
<p><span>March 9, 2016</span></p>
</li>
<li>
</ul>
</div>
</div>
</div>
<br>
<br>
<br>
<br>
<br>
<br>
via Wait But Why https://waitbutwhy.com<br>
<br>
October 26, 2018 at 08:55AM
